docs/releases/v2023.2.4.rst

```diff
@@ -0,0 +1,68 @@
+Gluon 2023.2.4
+==============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Sophos
+
+  - AP15
```

Comment:
Is the Sophos AP15 supposed to be on the list? I thought "broken" devices aren't listed?
https://github.com/freifunk-gluon/gluon/pull/3126#discussion_r1438419927